### PR TITLE
Branch to review -- Feature 1 

### PR DESCRIPTION
**Pull request by dcousette**
Please review this branch for inclusion into master. 

Thanks, 
DeShawn

### CODE REVIEW

File: spec/controllers/sessions_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      session[:user_id] = Fabricate(:user).id
+      get :new 
+      expect(response).to redirect_to home_path
+    end
+  end
+  
+  describe 'POST create' do 
+    context 'with valid credentials' do
+      before do 
+        # alice = Fabricate(:user)
```
Comment (brandoncc) [inline spec/controllers/sessions_controller_spec.rb]: Make sure you remove commented code.

File: spec/controllers/sessions_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      it 'does not put a user in the session' do
+        expect(session[:user_id]).to be_nil 
+      end
+      
+      it 'renders the new template' do 
+        expect(response).to render_template :new 
+      end 
+      
+      it 'sets the notice' do 
+        expect(flash[:danger]).not_to be_nil
```
Comment (brandoncc) [inline spec/controllers/sessions_controller_spec.rb]: You should always use the positive matcher when possible. Double negatives are bad.

``` ruby
expect(flash[:danger]).to be_present
```

File: spec/fabricators/review_fabricator.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Fabricator(:review) do 
+  rating { (1..5).to_a.sample }
+  content { Faker::Lorem.paragraph(3) }
```
Comment (brandoncc) [inline spec/fabricators/review_fabricator.rb:3]: I believe this returns an array, which you should join as well.

Comment (dcousette) [inline spec/fabricators/review_fabricator.rb:3]: To be clear, you're saying join the returned array of paragraphs, correct? So line 2 should be: 
content { Faker::Lorem.paragraph(3).join(" ") }  

--UPDATE---

I tried the line above and it caused my tests to fail.. so maybe you meant something else?

Comment (brandoncc) [inline spec/fabricators/review_fabricator.rb:3]: Ah, my apologies. `paragraphs` returns an array but `paragraph` returns a sentence. I thought they both returned paragraphs.

https://github.com/ffaker/ffaker/blob/master/lib/ffaker/lorem.rb#L38

Comment (dcousette) [inline spec/fabricators/review_fabricator.rb:3]: Ok, got it. Thanks!

**Pull request merged**